CIScripts/ContrailUtils.ps1

```diff
@@ -75,18 +75,40 @@ function Add-OpenContrailNetwork {
         }
     }
 
-    $RequestUrl = $OpenContrailUrl + "/virtual-networks"
+    $RequestUrl = $ContrailUrl + "/virtual-networks"
     $Response = Invoke-RestMethod -Uri $RequestUrl -Headers @{"X-Auth-Token" = $AuthToken} `
         -Method Post -ContentType "application/json" -Body (ConvertTo-Json -Depth 10 $Request) `
 
     return $Response.'virtual-network'.'uuid'
 }
 
-function Remove-OpenContrailNetwork {
-    Param ([Parameter(Mandatory = $true)] [string] $OpenContrailUrl,
+function Remove-ContrailVirtualNetwork {
+    Param ([Parameter(Mandatory = $true)] [string] $ContrailUrl,
            [Parameter(Mandatory = $true)] [string] $AuthToken,
-           [Parameter(Mandatory = $true)] [string] $NetworkUuid)
+           [Parameter(Mandatory = $true)] [string] $NetworkUuid,
+           [Parameter(Mandatory = $false)] [bool]  $Force = $false)
```

Comment:
You can just skip the whole `[Parameter...]`. The default is `Mandatory = $false`